dev-client/src/components/siteNotes/SiteNoteForm.tsx

```diff
@@ -0,0 +1,45 @@
+import {useTranslation} from 'react-i18next';
+import {FormInput} from 'terraso-mobile-client/components/common/Form';
+import {TextInput} from 'react-native';
+import {useRef, useEffect} from 'react';
+
+type SiteNoteFormProps = {
+  content: string;
+  onChangeContent: (text: string) => void;
+  onBlurContent: (e: any) => void;
+};
+
+export const SiteNoteForm = ({
+  content,
+  onChangeContent,
+  onBlurContent,
+}: SiteNoteFormProps) => {
+  const {t} = useTranslation();
+  const formInputRef = useRef<TextInput>(null);
+
+  useEffect(() => {
+    if (formInputRef.current) {
+      formInputRef.current.focus();
+    }
+  }, []);
+
+  return (
+    <FormInput
```

Comment:
question: What are the autoCapitalize / autoCorrect settings for this field?

Reply:
autoCapitalize is using the default 'sentences', and autoCorrect is using the default true.